includes/core/server/class-request.php

```diff
@@ -21,7 +21,7 @@
 /**
  * Base for requests to the WCPay server.
  */
-abstract class Request {
+abstract class Request{
```

Comment:
I'm not sure how this change got added, let's fix the whitespace.

Reply:
Fixed here: https://github.com/Automattic/woocommerce-payments/pull/7314/commits/b72ea986fdd961f8586f2173aa7c8bf0087cbeb8